### ISSUE TITLE
27304689: MKMapView accessibilityElementsHidden has no effect

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#15346]
#### Description

Summary:
The accessibilityElementsHidden property on MKMapView appears to have no effect. Setting it to true still results in VoiceOver navigating over the various elements within the map view.

Steps to Reproduce:
1. Create an instance of MKMapView and attach it to a view hierarchy
2. Set accessibilityElementsHidden to true on the MKMapView instance
3. Load some coordinate (somewhere with roads)
4. While the view is visible, enable VoiceOver and tap within the bounds of the map view

Expected Results:
VoiceOver cursor should not navigate the map elements, the map view should be skipped entirely. 

Actual Results:
The various elements (roads and landmarks) of the map view are navigable via VoiceOver.
## 

Product Version: 9.3
Created: 2016-07-12T18:29:55.690970
Originated: 2016-07-12T14:29:00
Open Radar Link: http://www.openradar.me/27304689
